tests/labhub_test.py

```diff
@@ -180,13 +183,14 @@ def test_create_issue_cmd(self):
 
         # not a member of org
         self.mock_team.is_member.return_value = False
-        testbot_public.assertCommand(
-            textwrap.dedent('''\
-                !new issue repository this is the title
-                body
-            '''),
-            'You need to be a member of this organization to use this command.'
-        )
+        with self.assertRaises(queue.Empty):
+            testbot_public.assertCommand(
+                textwrap.dedent('''\
+                    !new issue repository this is the title
+                    body
+                '''),
+                'Here you go'
```

Comment:
ideally this test doesnt change, but if it has to, change this to garbage so it is obvious it shouldnt work